package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@google/gapic-generator-typescript",
```

Comment:
To just nit the name a little:
- We don't use the `@google` scope a lot.  Was this intentional?  Reason not to use `@google-cloud`?
- I think having `typescript` in the name is redundant.  Can it just be `gapic-generator`?

Reply:
1. Agree as we only generate apis for google-cloud
2. Will it be confusing with our gapic-genereator?

Reply:
I'm not too worried about that :)  The monolithic java gapic-generator isn't distributed on npm!

Reply:
I used "gapic-generator" in Python -- see https://pypi.org/project/gapic-generator/